packages/solidity/contracts/StakingRewards.sol

```diff
@@ -737,37 +738,158 @@ contract StakingRewards is ILiquidityProtectionEventsSubscriber, AccessControl,
     }
 
     /**
-     * @dev updates pending rewards for a list of providers
+     * @dev updates pending rewards for a list of providers in a specific pool
      *
+     * @param poolToken the participating pool to update
      * @param providers owners of the liquidity
      */
-    function updateRewards(address[] calldata providers) external {
+    function updatePoolRewards(address[] calldata providers, IDSToken poolToken) external {
```

Comment:
Anyone can call?

Reply:
In this case, it was indeed not an issue, but please note that the newer `storePoolRewards` method is permissioned